cmd/forklift-controller/main.go

```diff
@@ -113,12 +118,75 @@ func main() {
 		log.Error(err, "unable to add kubevirt export APIs to scheme")
 		os.Exit(1)
 	}
+	if err := promv1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "unable to add Prometheus APIs to scheme")
+		os.Exit(1)
+	}
 	if err := template.Install(mgr.GetScheme()); err != nil {
 		log.Error(err, "proceeding without optional OpenShift template APIs")
 	}
 	if err := instancetype.AddToScheme(mgr.GetScheme()); err != nil {
 		log.Error(err, "proceeding without optional kubevirt instance type APIs")
 	}
+
+	openshift := os.Getenv("OPENSHIFT")
+	if openshift == "" {
+		openshift = "false"
+	}
+	// Clusters without OpenShift do not run OpenShift monitoring out of the box,
+	// and hence are not able to be registered to the monitoring services.
+	if openshift == "true" {
+		err = mgr.Add(manager.RunnableFunc(func(ctx context.Context) error {
+			log.Info("waiting for cache to sync")
```

Comment:
do you intend to keep this log? if you do, please add some context to the message